.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -85,6 +93,14 @@ jobs:
         timeout-minutes: 10
         run: bash tests/integration/kubernetes/gha-run.sh deploy-kata-tdx
 
+      - name: Install snapshotter
+        timeout-minutes: 10
+        run: bash tests/integration/kubernetes/gha-run.sh install-snaphostter
```

Comment:
is this a typo, should be `install-snapshotter`

Reply:
Fixed.